README.md

```diff
@@ -83,7 +90,7 @@ const userInput = { arbitrary: "json" };
 typeGuards.normandy_assertAddonRollbackArguments(userInput);
 // userInput has now been asserted to be of type `normandy.AddonRollbackArguments`.
 // If it wasn't the previous line would have thrown an error.
-```
+````
```

Comment:
This should be three `` ` ``, I believe (if my hunch about the leaky fence above is correct).